app/controllers/redirections_controller.rb

```diff
@@ -20,9 +20,8 @@ def previous
   private
```

Comment:
This diff is best viewed with whitespace ignored: https://github.com/hotline-webring/hotline-webring/pull/373/files?diff=unified&w=1